crops/crop.go

```diff
@@ -115,3 +124,36 @@ func (c Crop) ComputeConsequences(event interface{}) consequences.Results {
 	r := consequences.Results{IsTable: false, Result: ret}
 	return r
 }
+func (c Crop) computeImpactedCase(e hazards.ArrivalandDurationEvent) float64 {
+	// Determine damage percent based on damage dur curve and event dur
+	perdmg := c.lossFunction.ComputeDamagePercent(e)
+	fmt.Println("damage percent is ", perdmg)
+	croploss := (perdmg / 100) * c.GetValuePerOutputUnit()
+	fmt.Println("Crop loss is : ", croploss)
+	// value added to field before loss by production
+	fmt.Println("total pruduction costs are:", c.productionFunction.productionCostLessHarvest)
+	fmt.Println("fixed costs are: ", c.productionFunction.GetfixedCosts())
+	loss := croploss + c.productionFunction.productionCostLessHarvest
+	fmt.Println("total loss is:", loss)
+	if loss > c.valuePerOutputUnit {
+		// Throw some error
+	}
+	return 10
```

Comment:
this return needs to be fixed.

Reply:
keeping 10 until i update the test to reflect correct result

Reply:
hand calced proper return and updated